Synchronize Advisory Committee representative and Consortium Member Representative

frivoal opened the pull request:
See https://github.com/w3c/AB-memberonly/issues/247


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/937.html" title="Last updated on Jan 20, 2026, 9:48 AM UTC (23b655b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/process/937/7daee46...frivoal:23b655b.html" title="Last updated on Jan 20, 2026, 9:48 AM UTC (23b655b)">Diff</a>

frivoal committed:
Synchronize Advisory Committee representative and Consortium Member Representative

See https://github.com/w3c/AB-memberonly/issues/247

Co-authored-by: fantasai <fantasai.bugs@inkedblade.net>

frivoal committed:
punctuation

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

NameNoQuality commented on index.bs:
```diff
@@ -761,7 +761,7 @@ Participation in the Advisory Committee</h4>
 	For the purposes of the W3C Process,
 	this person also serves as the organization's [=Advisory Committee Representative=].
-	When a [=Member Association=] joins W3C
+	When a [=Member Association=] joins W3C,
```
#912

NameNoQuality replied:
@aungko.use@gmail.com

tantek reviewed (approved):
This seems like a reasonable step forward.

I'd request that the editor consider the impact of these changes on the "alternate representative" role to make sure that role still works as expected/desired. I trust the editor's judgment on this interaction.

frivoal commented:
The proposed text has mentions member associations, and therefore has a dependency on https://github.com/w3c/board/issues/249

TallTed commented on index.bs:
```diff
@@ -766,13 +766,17 @@ Participation in the Advisory Committee</h4>
-	(see “<a href="https://www.w3.org/membership/join/">How to Join W3C</a>” [[JOIN]]),
-	it <em class="rfc2119">must</em> name its Advisory Committee representative as part of their Membership Agreement.
+	As defined in the W3C Bylaws,
+	each W3C [=Member=] organization that is not a [=Member Association=] (known as a “<i>Consortium Member</i>” in the Bylaws)
```
The existing parenthetical structure makes it seem that _a `Member Association` is known as a `Consortium Member` in the Bylaws_, rather than _a W3C Member organization that is not a `Member Association` is known as a `Consortium Member` in the Bylaws_.
```suggestion
	each “<i>Consortium Member</i>” (that is, each W3C [=Member=] organization that is not a [=Member Association=])
```

TallTed replied:
or
```suggestion
	each W3C [=Member=] organization that is not a [=Member Association=] (that is, each organization the Bylaws would refer to as a “<i>Consortium Member</i>”)
```

frivoal replied:
Good catch. I think I lean towards your first suggestion, but both seam reasonable to me. Any other opinions?

TallTed commented on index.bs:
```diff
@@ -766,13 +766,17 @@ Participation in the Advisory Committee</h4>
+	As defined in the W3C Bylaws,
+	each W3C [=Member=] organization that is not a [=Member Association=] (known as a “<i>Consortium Member</i>” in the Bylaws)
+	has one “<i>Consortium Member Representative</i>” [[!BYLAWS]].
+	For the purposes of the W3C Process,
```
```suggestion
	For purposes of the W3C Process,
```

TallTed reviewed (changes requested):
Language tweaks for clarity.

frivoal committed:
Fix syntax

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

tantek reviewed (approved):
Looks good with either of the suggested fixes by @TallTed.

frivoal committed:
Adjust phrasing

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

frivoal committed:
Reduce coupling with the Bylaws

The Process only needs to concern itself with the fact that the bylaws
define that Consortium Members have Consortium Member Representatives,
and that some Members (in the sense of the Process) do not qualify to be
Consortium Members (and thus don't have a C.M.Rep.). It does not need to
concern itself with why the bylaws exclude some organisations from being
Consortium Members, and therefore reducing the interdependency is
desirable: this allows the bylaws to evolve the criteria if needed,
without needing any revision in the Process.

frivoal committed:
Clarification on alt-ACRep and (the lack of) bylaws implications
